cgi-bin/LJ/Support.pm

```diff
@@ -1152,15 +1153,39 @@ sub work {
         }->{$rtype};
 
         # build body
-        $body = LJ::Lang::ml( "support.email.notif.update.body3", {
+        $body = LJ::Lang::ml( "support.email.notif.update.body4", {
                 sitename => $LJ::SITENAMESHORT,
                 category => $sp->{_cat}{catname},
                 subject => $sp->{subject},
-                type => $response_type,
                 username => LJ::trim( $show_name ),
                 url => "$LJ::SITEROOT/support/see_request?id=$spid",
-                text => $resp
+                type => $response_type
             } );
+        if ( $faqid ) {
+            # need to set up $lang
+            my $lang;
+            my $u = LJ::load_userid( $posterid ) if $posterid;
```

Comment:
So, `my $x = foo if $bar;` is a trap in some versions of Perl, so we don't allow it. See: http://wiki.dreamwidth.net/wiki/index.php/Dev_Programming_Guidelines#Postfix_conditionals

You can just do `my ( $lang, $u );` above and then `$u = LJ::load_userid( $posterid ) if $posterid;` in this line.